util/Migrator/DbScripts/2025-12-11_00_User_UpdateKeyConnectorUserKey.sql

```diff
@@ -0,0 +1,35 @@
+IF OBJECT_ID('[dbo].[User_UpdateKeyConnectorUserKey]') IS NOT NULL
+BEGIN
+    DROP PROCEDURE [dbo].[User_UpdateKeyConnectorUserKey]
+END
+GO
+
+CREATE PROCEDURE [dbo].[User_UpdateKeyConnectorUserKey]
```

Comment:
```suggestion
CREATE OR ALTER PROCEDURE [dbo].[User_UpdateKeyConnectorUserKey]
```

We normally use `CREATE OR ALTER PROCEDURE` for SP migrations unless something has changed recently.